.gitignore

```diff
@@ -45,6 +45,7 @@ htmlcov/
 
 pip-wheel-metadata/
 
-parfive/version.py
 parfive/tests/.ipynb_checkpoints/
 parfive/tests/predicted-sunspot-radio-flux.txt
+parfive/_version.py
+coverage.xml
```

Comment:
Won't need to undo this?

Reply:
it's still saving `_version.py` it's just only putting the install time number in there now